targets/TARGET_NORDIC/TARGET_NRF5x/us_ticker.c

```diff
@@ -100,6 +100,9 @@ void us_ticker_init(void)
 
     nrf_timer_task_trigger(NRF_TIMER1, NRF_TIMER_TASK_START);
 
+    /* Bug fix. First value can't be trusted. */
+    nrf_timer_task_trigger(NRF_TIMER1, NRF_TIMER_TASK_CAPTURE1);
```

Comment:
If this is somehow related to [one of the known anomalies](http://infocenter.nordicsemi.com/topic/com.nordic.infocenter.nrf52832.Rev2.errata/dita/errata/nRF52832/Rev2/latest/anomaly_832_201.html?cp=2_1_1_0_1_52), it may be worth adding a reference.

Reply:
I don't think it is. I just noticed some very odd behavior on my boards.